concepticondata/conceptlists/Ratcliffe-2020-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Ratcliffe-2020-100-1	1	ALL	98	ALL
+Ratcliffe-2020-100-2	2	ASH	646	ASH
+Ratcliffe-2020-100-3	3	BARK	1204	BARK
+Ratcliffe-2020-100-4	4	BELLY	1251	BELLY
+Ratcliffe-2020-100-5	5	BIG	1202	BIG
+Ratcliffe-2020-100-6	6	BIRD	937	BIRD
+Ratcliffe-2020-100-7	7	BITE	1403	BITE
+Ratcliffe-2020-100-8	8	BLACK	163	BLACK
+Ratcliffe-2020-100-9	9	BLOOD	946	BLOOD
+Ratcliffe-2020-100-10	10	BONE	1394	BONE
+Ratcliffe-2020-100-11	11	BREAST	1402	BREAST
+Ratcliffe-2020-100-12	12	BURN (VT	141	BURN (SOMETHING)
+Ratcliffe-2020-100-13	13	CLAW	72	CLAW
+Ratcliffe-2020-100-14	14	CLOUD	1489	CLOUD
+Ratcliffe-2020-100-15	15	COLD	1287	COLD
+Ratcliffe-2020-100-16	16	COME	1446	COME
+Ratcliffe-2020-100-17	17	DIE	1494	DIE
+Ratcliffe-2020-100-18	18	DOG	2009	DOG
+Ratcliffe-2020-100-19	19	DRINK	1401	DRINK
+Ratcliffe-2020-100-20	20	DRY	1398	DRY
+Ratcliffe-2020-100-21	21	EAR	1247	EAR
+Ratcliffe-2020-100-22	22	EARTH, DIRT	2158	EARTH (WET SUBSTANCE)
```

Comment:
2158 is retired and should be changed to 1228. (perhaps you could fix this for Wang-2004-471 while you're at it?)